lib/runtime/wazero/instance.go

```diff
@@ -108,7 +108,7 @@ func NewInstance(code []byte, cfg Config) (instance *Instance, err error) {
 
 	_, err = rt.NewHostModuleBuilder("env").
 		// values from newer kusama/polkadot runtimes
-		ExportMemory("memory", 23).
+		ExportMemory("memory", 2070).
```

Comment:
First of all, no magic variables please.  This should be const var with specific name. 
Secondly, if this value static it should be 65k. Please refer to our slack thread for more details.